azure-pipelines/build.yml

```diff
@@ -23,29 +23,40 @@ jobs:
     displayName: 'Install Node.js'
   - script: npm ci
     displayName: 'npm ci'
-  - script: npm run build-nomaps
-    displayName: 'npm run build-nomaps'
-  - powershell: .\package.ps1
-  - task: NuGetCommand@2
+  - script: npm run build
+    displayName: 'npm run build'
+  - script: npm run webpack
+    displayName: 'npm run webpack'
```

Comment:
> Why did we drop the -nomaps option

I assume someone added "-nomaps" to prevent cluttering up the package with "map" files, but they're never included anyway since we explicitly copy each file we want to a staging directory before packaging.

> was webpack always part of this pipeline?

Yes, it was in the `package.ps1` script before (which felt pretty hidden to me)